server/plugin.go

```diff
@@ -595,3 +595,24 @@ func (p *Plugin) HasGroupHook(ctx context.Context, user *gitlab.UserInfo, namesp
 
 	return found, err
 }
+
+// getUsername returns the GitLab username for a given Mattermost user,
+// if the user is connected to GitLab via this plugin.
+// Otherwise it return the Mattermost username. It will be escaped via backticks.
```

Comment:
```suggestion
// Otherwise it returns the Mattermost username. It will be escaped via backticks.
```
What do you mean by "It will be escaped via backticks"?